docs/docs/tracing/minimal-sdk.mdx

```diff
@@ -0,0 +1,215 @@
+---
+description: MLflow Tracing SDK (mlflow-tracing) is a lightweight package that includes only the essential functionality for tracing and monitoring your GenAI applications.
+sidebar_position: 5
+sidebar_label: Lightweight SDK
+---
+
+import { Card, CardGroup, TitleCard } from "@site/src/components/Card";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Lightweight Tracing SDK Optimized for Production Usage
+
+MLflow offers a lightweight tracing SDK package called `mlflow-tracing` that includes only the essential functionality for tracing and monitoring your GenAI applications. This package is designed for production environments where minimizing dependencies and deployment size is crucial.
+
+## Why Use the Lightweight SDK?
+
+
+<CardGroup cols={2}>
+    <TitleCard
+        title="🏎️ Faster Deployment"
+        description="The package size and dependencies are significantly smaller than the full MLflow package, allowing for faster deployment times in dynamic environments such as Docker containers, serverless functions, and cloud-based applications."
+    />
+    <TitleCard
+        title="🔧 Simplified Dependency Management"
+        description="A smaller set of dependencies means less work keeping up with dependency updates, security patches, and breaking changes from upstream libraries."
+    />
+    <TitleCard
+        title="📦 Portability"
+        description="With the less number of dependencies, MLflow Tracing can be easily deployed across different environments and platforms, without worrying about compatibility issues."
+    />
+    <TitleCard
+        title="🔒 Fewer Security Risks"
+        description="Each dependency potentially introduces security vulnerabilities. By reducing the number of dependencies, MLflow Tracing minimizes the attack surface and reduces the risk of security breaches."
+    />
+</CardGroup>
+
+## Installation
+
+Install the lightweight SDK using pip:
+
+```bash
+pip install mlflow-tracing
+```
+
+:::warning
+
+Do not install the full `mlflow` package together with the lightweight SDK, as this may cause version conflicts.
```

Comment:
```suggestion
Do not install the full `mlflow` package together with the lightweight `mlflow-tracing` SDK, as this may cause version conflicts and namespace resolution issues.
```